climada/hazard/tc_tracks.py

```diff
@@ -1127,6 +997,9 @@ def equal_timestep(self, time_step_h=1, land_params=False):
         land_params : bool, optional
             If True, recompute `on_land` and `dist_since_lf` at each node. Default: False.
         """
+        if pool is not None:
+            self.pool = pool
```

Comment:
Could this potentially create issues if the object already has a pool? Say for instance we have a `TCTracks` object with a pool, which is used and not properly closed before calling `equal_timestep` with a new pool. Would that not be a problem?

I am not sure whether it is a good idea to add `pool` as parameter of `equal_timestep` given that `TCTracks` objects already have the attribute `pool`...